docs/configuration.asciidoc

```diff
@@ -1045,6 +1045,39 @@ you must instead set the `LogLevel` for the internal APM logger under the `Loggi
 | `Error`                 | String
 |============
 
+[float]
+[[config-trace-context-ignore-sampled-false]]
+==== `TraceContextIgnoreSampledFalse`
+
+[IMPORTANT]
+--
+..NET 5 applications set the https://www.w3.org/TR/trace-context/[W3C Trace Context] headers for outgoing HTTP requests by default, but with the traceparent sampled flag set to `false`. If the application has an active agent, the agent will ensure that the sampled flag is propagated appropriately. If the application does not have an active agent however, and the application calls another application (the callee) that does have an active agent, the propagation of a sampled flag of `false` results in no sampled traces in the callee application.
```

Comment:
It seems we need an additional `.` to render `.NET` in asciidoc when it's the 1. word in the section.

Reply:
This looks like it renders the paragraph in red italics in asciidoctor.js

![image](https://user-images.githubusercontent.com/208231/121973319-53e71480-cdc0-11eb-91cb-9cc9930aa26d.png)

An attribute entry can be used

```suggestion
:dot: .

{dot}NET 5 applications set the https://www.w3.org/TR/trace-context/[W3C Trace Context] headers for outgoing HTTP requests by default, but with the traceparent sampled flag set to `false`. If the application has an active agent, the agent will ensure that the sampled flag is propagated appropriately. If the application does not have an active agent however, and the application calls another application (the callee) that does have an active agent, the propagation of a sampled flag of `false` results in no sampled traces in the callee application.
```

which renders as

![image](https://user-images.githubusercontent.com/208231/121973417-842eb300-cdc0-11eb-8072-af4cc3a14102.png)

Reply:
Can ignore the suggestion here as it's included in the next suggestion